core/templates/pages/exploration-player-page/layout-directives/audio-bar.component.spec.ts

```diff
@@ -161,6 +164,14 @@ describe('Audio Bar Component', () => {
     expect(result).toBe(false);
   });
 
+  it('should return true if the selected language is RTL', () => {
```

Comment:
add a test for the false case too

Reply:
HI @gp201 , I updated the PR with your comments. Thanks!